<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QWidget" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>menu</string>
  </property>
  <widget class="QLabel" name="cpuNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

font: 700 9pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>CPU : </string>
   </property>
  </widget>
  <widget class="QLabel" name="gpuNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

font: 700 9pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>GPU :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>23</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>CPU</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>RAM</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>GPU</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>VRAM</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="cbar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
    border: 1px solid #555;
    border-radius: 5px;

    text-align: center;
    color: white;
}

QProgressBar::chunk {
    background-color: #3daee9;   /* 进度条颜色 */
    border-radius: 5px;
}</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="crbar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
    border: 1px solid #555;
    border-radius: 5px;
    text-align: center;
    color: white;
}

QProgressBar::chunk {
    background-color: rgb(187, 28, 255);   
    border-radius: 5px;
}</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="gbar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
    border: 1px solid #555;
    border-radius: 5px;

    text-align: center;
    color: white;
}

QProgressBar::chunk {
    background-color: #2ecc71;   /* 进度条颜色 */
    border-radius: 5px;
}</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="grbar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
    border: 1px solid #555;
    border-radius: 5px;

    text-align: center;
    color: white;
}

QProgressBar::chunk {
    background-color: #3daee9;   /* 进度条颜色 */
    border-radius: 5px;
}</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cpuVal">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>78%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="cpuramVal">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>3.5G</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="gpuVal">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>78%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="gpuramVal">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>4.0G</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>431</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(0,0,0);
border-radius:10px;
background-color: rgba(235, 235, 235, 155);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgba(255,0,0,155);
border-radius:10px;
background-color: rgba(255, 0, 0,15);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgba(0, 155, 0, 155);
border-radius:10px;
background-color: rgba(0, 215, 0, 15);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgba(5, 197, 255, 155);
border-radius:10px;
background-color: rgba(0, 0, 255, 15);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgba(255, 131, 6, 155);
border-radius:10px;
background-color: rgba(255, 172, 28, 15);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_5</zorder>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>cpuNameLabel</zorder>
  <zorder>gpuNameLabel</zorder>
  <zorder>label1</zorder>
  <zorder>label2</zorder>
  <zorder>label3</zorder>
  <zorder>label4</zorder>
  <zorder>cbar</zorder>
  <zorder>crbar</zorder>
  <zorder>gbar</zorder>
  <zorder>grbar</zorder>
  <zorder>cpuVal</zorder>
  <zorder>cpuramVal</zorder>
  <zorder>gpuVal</zorder>
  <zorder>gpuramVal</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
